docker_manager.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# The DockerManager class provides methods for managing Docker images and containers.
+# It allows to download an image, run a container, multiple containers and remove all the containers and images.
+# The DockerManager class provides methods for managing Docker images and containers.
+# It allows to download an image, run a container, multiple containers and remove all the containers and images.
+class DockerManager
+  def initialize(data, image_base_name)
+    @data = data
+    @image_base_name = image_base_name
+  end
+
+  def download_image(image_name)
+    system("docker pull #{@image_base_name}:#{image_name}")
```

Comment:
the `image_name:version` structure is usually used to load an image

Reply:
fix in new commit